### PR TITLE
[SCB-2475] migrate to junit5 transports swagger part 1

### PR DESCRIPTION
**Pull request by Shoothzj**

### CODE REVIEW

File: transports/transport-highway/src/main/java/org/apache/servicecomb/transport/highway/HighwayServerVehicle.java [hunk 1]
```diff
@@ -31,8 +31,8 @@
 import io.vertx.core.Promise;
 import io.vertx.core.Vertx;
 
-public class HighwayServerVerticle extends AbstractVerticle {
-  private static final Logger LOGGER = LoggerFactory.getLogger(HighwayServerVerticle.class);
+public class HighwayServerVehicle extends AbstractVerticle {
```
Comment (liubao68) [inline transports/transport-highway/src/main/java/org/apache/servicecomb/transport/highway/HighwayServerVehicle.java:34]: Why rename this class?

File: transports/transport-rest/transport-rest-vertx/src/main/java/org/apache/servicecomb/transport/rest/vertx/RestBodyHandler.java [hunk 2]
```diff
@@ -90,6 +91,11 @@ private RestBodyHandler(boolean handleFileUploads, String uploadDirectory) {
     setUploadsDirectory(uploadDirectory);
   }
 
+  @VisibleForTesting
```
Comment (liubao68) [inline transports/transport-rest/transport-rest-vertx/src/main/java/org/apache/servicecomb/transport/rest/vertx/RestBodyHandler.java:94]: we can set `VisibleForTesting` default to package visible, how do you think so?

File: transports/transport-highway/src/main/java/org/apache/servicecomb/transport/highway/HighwayServerVehicle.java (hunk 1 above)
Comment (Shoothzj) [reply transports/transport-highway/src/main/java/org/apache/servicecomb/transport/highway/HighwayServerVehicle.java:34]: @liubao68 That `Verticle` might be a typo

File: transports/transport-rest/transport-rest-vertx/src/main/java/org/apache/servicecomb/transport/rest/vertx/RestBodyHandler.java (hunk 2 above)
Comment (Shoothzj) [reply transports/transport-rest/transport-rest-vertx/src/main/java/org/apache/servicecomb/transport/rest/vertx/RestBodyHandler.java:94]: @liubao68 what do you mean, remove the `public` modifier?

File: swagger/swagger-generator/generator-core/src/test/java/org/apache/servicecomb/swagger/generator/core/TestSwaggerGenerator.java [hunk 3]
```diff
@@ -26,6 +26,7 @@
 import org.junit.Assert;
```
Comment (david6969xin) [inline swagger/swagger-generator/generator-core/src/test/java/org/apache/servicecomb/swagger/generator/core/TestSwaggerGenerator.java:26]: remove the unused improt is better, you can find this in several modified files, so you can double check

Comment (Shoothzj) [reply swagger/swagger-generator/generator-core/src/test/java/org/apache/servicecomb/swagger/generator/core/TestSwaggerGenerator.java:26]: @david6969xin Great. We can enable the unused import check on checkstyle. Can you work on this?

Comment (Shoothzj) [reply swagger/swagger-generator/generator-core/src/test/java/org/apache/servicecomb/swagger/generator/core/TestSwaggerGenerator.java:26]: I will fix it.

File: transports/transport-rest/transport-rest-vertx/src/main/java/org/apache/servicecomb/transport/rest/vertx/RestBodyHandler.java (hunk 2 above)
Comment (liubao68) [reply transports/transport-rest/transport-rest-vertx/src/main/java/org/apache/servicecomb/transport/rest/vertx/RestBodyHandler.java:94]: yes

File: transports/transport-highway/src/main/java/org/apache/servicecomb/transport/highway/HighwayServerVehicle.java (hunk 1 above)
Comment (liubao68) [reply transports/transport-highway/src/main/java/org/apache/servicecomb/transport/highway/HighwayServerVehicle.java:34]: `Verticle` is correct, not `Vehicle`

**Comment by Shoothzj** [comment]
@liubao68 @david6969xin PTAL again. I have fixed all addressed problems.

**Review by liubao68 — APPROVED** [review]

**Pull request merged**